release_creation/github_client/releases.py

```diff
@@ -21,13 +21,20 @@ def get_github_client() -> Github:
     return Github(_GH_ACCESS_TOKEN)
 
 
-def normalize_version(version: Version) -> Version:
+def _normalize_version_tags(version:Version) -> Version:
+    if version.prerelease or version.build:
+        version.prerelease = ["pre"]
+    if version.build:
+        version.build = []
```

Comment:
Are these lists for comparisons later because that's what github returns when we're checking the latest?

Reply:
The `Version` class assumes these inputs are iterators so if we pass it "pre" it's going to generate semantic version like `1.0.0.p.r.e`